src/Cardano/CoinSelection/LargestFirst.hs

```diff
@@ -48,7 +57,7 @@ import qualified Data.Map.Strict as Map
 --      sorted into /descending order of coin value/.
 --
 --      The /head/ of the list is always the remaining UTxO entry with the
---      /greatest coin value/.
+--      /largest coin value/.
```

Comment:
This might seem like a trivial change, but the wording is now consistent with the name of the algorithm.